### PR TITLE
Add a check rule for password strength

### PR DESCRIPTION
**Pull request by roxanemace**
**Asana ticket**: https://app.asana.com/0/1141396442496489/1163374331637444

### CODE REVIEW

File: src/components/form/fields/passwordPolicyRules.jsx [hunk 1]
```diff
@@ -65,9 +65,17 @@ const PasswordPolicyIntro = withI18n(withTheme(styled(({ i18n, className }) =>
     margin-bottom: ${props => props.theme.get('spacing')}px;
 `));
 
-export const PasswordPolicyRules = withTheme(styled(({ className, ...props }) =>
+export const PasswordPolicyRules = withI18n(withTheme(styled(({ className, i18n, ...props }) =>
     <div className={className}>
+        {props.minStrength !== 0 && <PasswordPolicyRule>
```
Comment (StanWarz) [inline src/components/form/fields/passwordPolicyRules.jsx:70]: '{props.minStrength &&' should be enough to assert the condition.

**Review by StanWarz — APPROVED** [review]

File: src/components/form/fields/passwordPolicyRules.jsx (hunk 1 above)
Comment (roxanemace) [reply src/components/form/fields/passwordPolicyRules.jsx:70]: In fact, I don't want to display the check rule if the strength is set to "None". Your suggestion doesn't do that :)

Comment (StanWarz) [reply src/components/form/fields/passwordPolicyRules.jsx:70]: Edited yes my bad!

Comment (roxanemace) [reply src/components/form/fields/passwordPolicyRules.jsx:70]: Thanks for the suggestion anyway :)

**Review by axuan25 — APPROVED** [review]

**Pull request merged**